src/metadataBlocks/index.ts

```diff
@@ -1,13 +1,23 @@
 import { GetMetadataBlockByName } from './domain/useCases/GetMetadataBlockByName'
 import { MetadataBlocksRepository } from './infra/repositories/MetadataBlocksRepository'
 import { GetCollectionMetadataBlocks } from './domain/useCases/GetCollectionMetadataBlocks'
+import { GetAllFacetableMetadataFields } from './domain/useCases/GetAllFacetableMetadataFields'
+import { MetadataFieldInfosRepository } from './infra/repositories/MetadataFieldInfosRepository'
 
 const metadataBlocksRepository = new MetadataBlocksRepository()
+const metadataFieldInfosRepository = new MetadataFieldInfosRepository()
 
 const getMetadataBlockByName = new GetMetadataBlockByName(metadataBlocksRepository)
 const getCollectionMetadataBlocks = new GetCollectionMetadataBlocks(metadataBlocksRepository)
+const getAllFacetableMetadataFields = new GetAllFacetableMetadataFields(
+  metadataFieldInfosRepository
+)
 
-export { getMetadataBlockByName, getCollectionMetadataBlocks }
+export {
+  getMetadataBlockByName,
+  getCollectionMetadataBlocks,
+  getAllFacetableMetadataFields as getAllFacetableDatasetFields
```

Comment:
Just asking, why renaming to `getAllFacetableDatasetFields` ?

Reply:
I think something happened when renaming the use case through the IDE. Fixed. Thanks for noticing!